tests/js/client/shell/api/multi/coordinator-metrics-cluster.js

```diff
@@ -110,8 +111,10 @@ function checkCoordinators(coordinators, mode) {
   assertTrue(coordinators.length > 1);
   for (let i = 1; i < coordinators.length; i++) {
     let c = coordinators[i];
-    let txt = getAllMetric(c, mode);
-    checkRawMetrics(txt, false);
+    checkIndexMetrics(function() {
```

Comment:
Please remove checkIndexMetrics wrapping in 106 line
Also your include unnecessary now